Tolocalfile propagates `saved_relPath` to posting

`saved_relPath` and `saved_baseUrl` can get propogated to the posted message downstream when `accept.tolocalfile` is being called.

The `tolocalfile` plugin doesn't add `saved_relPath` to the `_deleteOnPost` and when a path is longer then 255 characters, sarracenia drops the message on post.

```
ddi-cmc-ops03: 2026-02-20 15:35:34,893 [ERROR] sarracenia.moth.amqp putNewMessage message header saved_relPath too long, dropping
ddi-cmc-ops03: 2026-02-20 15:35:34,893 [DEBUG] sarracenia.diskqueue put DEBUG add to new file diskqueue_post_retry_001.new {'_format': 'v02', '_deleteOnPost': {'subtopic', 'post_format', 'data_checksum', 'new_file', 'new_subtopic', 'new_relPath', 'new_baseUrl', 'report', 'onfly_checksum', 'publisher_index', 'old_format', 'posts', '_mask_index', 'old_relPath', '_format', 'new_path', 'local_offset', '_matches', 'new_inflight_path', 'old_baseUrl', 'new_dir', 'old_subtopic', 'exchange', 'post_failures', '_isRetry'}, ... 'saved_relPath': '20260220/MSC-DMS-OP/GSWOB/partners/carletonu_cryologger/02/academic.observation.atmospheric.surface_weather.carletonu_cryologger-1.0-ascii.product_generic_swob-xml-2.0.20260220020001.carletonu_tst.tst.orig:DMS:CMC:SWOB_CARLETONU_CRYOLOGGER:XML:20260220021006',  ...
```

I think if we add `saved_relPath` and `saved_baseUrl`  to the `_deleteOnPost` at the end of the plugin it should fix the problem.

I think there's also other issues with the plugin so it's probably worth to fix those too.

Another work around is to just not use the plugin if possible.

--- a/sarracenia/flowcb/accept/tolocalfile.py
+++ b/sarracenia/flowcb/accept/tolocalfile.py
@@ -90,6 +90,7 @@ class ToLocalFile(FlowCB):
 
             message['saved_baseUrl'] = message['baseUrl']
             message['saved_relPath'] = message['relPath']
+            message['_deleteOnPost'] |= set(['saved_baseUrl', 'saved_relPath'])
             message['baseUrl'] = 'file:'
 
             if self.o.baseDir and not message['relPath'].startswith(self.o.baseDir):

--- a/sarracenia/flowcb/rename/raw2bulletin.py
+++ b/sarracenia/flowcb/rename/raw2bulletin.py
@@ -152,6 +152,7 @@ class Raw2bulletin(FlowCB):
             # Assign a default value for messages not coming from AM
             if 'isProblem' not in msg:
                 msg['isProblem'] = False
+                msg['_deleteOnPost'] |= set(['isProblem'])
 
 
             # Rename file with data fetched

